### PR TITLE
Add crun

### PR DESCRIPTION
**Pull request by mbargull**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/crun/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+autoreconf -fis
+./configure \
+    --prefix="${PREFIX}" \
+    --disable-systemd \
+    --disable-static \
+    --enable-shared
+make
```
Comment (chrisburr) [inline recipes/crun/build.sh:9]:
```suggestion
make -j${CPU_COUNT}
```

**Comment by mbargull** [comment]
@chrisburr, Azure didn't seem to like https://github.com/conda-forge/staged-recipes/commit/eaf231b1592532d02aa7cce78c55667a68cce3ff and gives
```
Pipeline configuration error:
/.azure-pipelines/azure-pipelines-feedstock-creation.yml (Line: 9, Col: 3): Unexpected value 'environment'
```

**Comment by chrisburr** [comment]
Yeah, sorry about that. The CI is back in a functional state again 😄

**Comment by mbargull** [comment]
All good, thanks for working on it -- and also thanks for the thorough reviews! Very much appreciated!

**Pull request merged**